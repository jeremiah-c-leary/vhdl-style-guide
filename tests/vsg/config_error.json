{
  "file_list":[,
    "tests/vsg/entity1.vhd"
  ]
}
