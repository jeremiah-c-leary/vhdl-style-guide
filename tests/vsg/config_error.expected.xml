<?xml version="1.0" ?>
<testsuite errors="0" hostname="Rimmer" failures="1" timestamp="2018-12-07T03:10:28" tests="1" time="0" name="vhdl-style-guide">
  <properties>
  </properties>
  <testcase name="tests/vsg/config_error.json" time="0">
    <failure type="Failure">
      Invalid JSON format.  Review configuration for errors.
    </failure>
  </testcase>
  <system-out>
  </system-out>
  <system-err>
  </system-err>
</testsuite>
